components/discord_bot/actions/create-guild-channel/create-guild-channel.mjs

```diff
@@ -0,0 +1,159 @@
+import common from "../common.mjs";
+import constants from "../../constants.mjs";
+
+const { discord } = common.props;
+
+export default {
+  ...common,
+  key: "discord_bot-create-guild-channel",
+  name: "Create Guild Channel",
+  description: "Create a new channel for the guild. [See the docs here](https://discord.com/developers/docs/resources/guild#create-guild-channel)",
+  type: "action",
+  version: "0.0.1",
+  props: {
+    discord,
+    guildId: {
+      ...common.props.guildId,
+    },
+    name: {
```

Comment:
Include `common.props`, and you can leave out `discord` and `guildId` since they're already included in `common.props`.
```
props: {
  ...common.props,
  name: {
```